file-content-extraction/operations.py

```diff
@@ -0,0 +1,92 @@
+from connectors.core.connector import get_logger, ConnectorError
+from connectors.cyops_utilities.builtins import download_file_from_cyops
+from connectors.cyops_utilities.builtins import extract_artifacts
+
+import os
+import logging
+import tika
+import json
+import tika.tika
+from tika import parser
+from tika import language
+from tika import translate
+from tika import config as tika_config
+
+logger = get_logger('text_extractor')
+#logger.setLevel(logging.DEBUG)
+
+#const
+TMP_PATH = '/tmp/'
+CONNECTOR_DIR = os.path.dirname(os.path.realpath(__file__))
+TIKA_SERVER_PATH = "/opt/cyops-search/bin/tika-server.jar"
+LANGUAGE_KEYS = CONNECTOR_DIR + '/language-keys'
+
+
+def _set_env(config):
+    ''' Set TIKA_SERVER_JAR env for online or offline modes '''
+    try:
+        os.environ['TIKA_TRANSLATOR'] = 'org.apache.tika.language.translate.GoogleTranslator'
+        if config.get('offline_mode') == True:
+            os.environ['TIKA_SERVER_JAR'] = TIKA_SERVER_PATH
+            logger.debug("Offline Mode, using local Tika file: {}".format(os.environ['TIKA_SERVER_JAR']))
+        else:
+            if os.environ.get('TIKA_SERVER_JAR') is not None:
+                del os.environ['TIKA_SERVER_JAR']
+            #TODO: implement language related actions
+            tika.tika.TikaServerClasspath = LANGUAGE_KEYS
```

Comment:
observed that this is creating tika.log and tika-server.log file under /tmp and the log level is INFO. change this to log level WARN and log files under /var/log/cyops/cyops-integrations/file-content-extraction

Reply:
loglevel set to Warning and log dir is set to: /var/log/cyops/cyops-integrations/file-content-extraction/